CHANGES.rst

```diff
@@ -1,8 +1,90 @@
+-  Error when using two image scrappers together `#579 <https://github.com/sphinx-gallery/sphinx-gallery/issues/579>`__
+-  Improve the junit xml `#576 <https://github.com/sphinx-gallery/sphinx-gallery/issues/576>`__
+-  Remove the note linking to the download section at the beginning of the example from latex/pdf output `#572 <https://github.com/sphinx-gallery/sphinx-gallery/issues/572>`__
+-  typing.TYPE_CHECKING is True at runtime in executed .py files `#570 <https://github.com/sphinx-gallery/sphinx-gallery/issues/570>`__
+-  How best to handle data files? `#565 <https://github.com/sphinx-gallery/sphinx-gallery/issues/565>`__
+-  ENH Add CSS for pandas dataframe `#544 <https://github.com/sphinx-gallery/sphinx-gallery/issues/544>`__
+
+**Merged pull requests:**
+
+-  MRG Remove last/first_notebook_cell redundancy `#626 <https://github.com/sphinx-gallery/sphinx-gallery/pull/626>`__ (`lucyleeow <https://github.com/lucyleeow>`__)
+-  DOC use # %% `#624 <https://github.com/sphinx-gallery/sphinx-gallery/pull/624>`__ (`lucyleeow <https://github.com/lucyleeow>`__)
+-  DOC capture repr in scraper section `#616 <https://github.com/sphinx-gallery/sphinx-gallery/pull/616>`__ (`lucyleeow <https://github.com/lucyleeow>`__)
+-  [MRG+1] DOC Improve doc of splitters and use in IDE `#615 <https://github.com/sphinx-gallery/sphinx-gallery/pull/615>`__ (`lucyleeow <https://github.com/lucyleeow>`__)
+-  DOC mention template `#613 <https://github.com/sphinx-gallery/sphinx-gallery/pull/613>`__ (`lucyleeow <https://github.com/lucyleeow>`__)
+-  recommend consistent use of one block splitter `#610 <https://github.com/sphinx-gallery/sphinx-gallery/pull/610>`__ (`mikofski <https://github.com/mikofski>`__)
+-  MRG, MAINT: Split CSS and add control `#599 <https://github.com/sphinx-gallery/sphinx-gallery/pull/599>`__ (`larsoner <https://github.com/larsoner>`__)
+-  MRG, MAINT: Update deps `#598 <https://github.com/sphinx-gallery/sphinx-gallery/pull/598>`__ (`larsoner <https://github.com/larsoner>`__)
+-  MRG, ENH: Link to methods and properties properly `#596 <https://github.com/sphinx-gallery/sphinx-gallery/pull/596>`__ (`larsoner <https://github.com/larsoner>`__)
+-  MAINT: Try to get nightly working `#592 <https://github.com/sphinx-gallery/sphinx-gallery/pull/592>`__ (`larsoner <https://github.com/larsoner>`__)
+-  Remove duplicate doc_solver entry in the API reference structure `#589 <https://github.com/sphinx-gallery/sphinx-gallery/pull/589>`__ (`kanderso-nrel <https://github.com/kanderso-nrel>`__)
+-  mention literalinclude in the doc `#582 <https://github.com/sphinx-gallery/sphinx-gallery/pull/582>`__ (`emmanuelle <https://github.com/emmanuelle>`__)
+-  MAINT: Bump AppVeyor to 3.7 `#575 <https://github.com/sphinx-gallery/sphinx-gallery/pull/575>`__ (`larsoner <https://github.com/larsoner>`__)
+-  Remove the note link to downloads from pdf output `#573 <https://github.com/sphinx-gallery/sphinx-gallery/pull/573>`__ (`benvial <https://github.com/benvial>`__)
```

Comment:
I usually try to go through all of these unclassified ones and add BUG or ENH labels to them, then they end up in the correct lists above instead of stuck in this "misc" category

Reply:
Under the `Merged pull requests` heading?

The `Closed issues` just seem to be a list of closed issues, do you classify these as well? They generally have a corresponding PR.

Reply:
Oh there are a few DOC changes but there is only BUG or ENH headings. Would you keep these under the 'misc'?

Reply:
Whoops I misunderstood you - yes I added a few labels but must have missed those PRs. I'm manually moving them but there are 2 categories DOC and Maint which don't fit under ENH or BUG.

Reply:
I usually add labels to both the issue and the PR so they get put in the right categories

For doc you can label them as doc but I think they just stay in the "misc" categories so not sure there is much point. It would be great if we could add a category in the changelog generator. You're welcome to try to do so, but last time I used it that thing it was pretty buggy and I ended up just being happy I got something usable

Reply:
Maint I'd classify as either ENH or BUG, but even unclassified is fine there

Reply:
> I usually add labels to both the issue and the PR so they get put in the right categories

Ah I actually moved some of the correctly categorised issues into the misc label ('closed issues') because I thought it was confusing to have a double up of the PR and corresponding issue but I can move these back!